src/esbmc/esbmc_parseoptions.cpp

```diff
@@ -1881,6 +1881,16 @@ bool esbmc_parseoptionst::process_goto_program(
       goto_coveraget tmp(ns, goto_functions);
       tmp.add_false_asserts();
     }
+
+    // From here on all changes should take phi-nodes into account
```

Comment:
Why should we take phi-nodes into account?

Reply:
I changed the comment. The idea is that  the phi-nodes locations are sensitive, so if the goto program changes the phinodes might become invalid.